pages/articles.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+    <div class="pa-5">
+        <div v-if="loading">
```

Comment:
can we remove this `div` and add this condition as ` <Loading :active="loading" v-if="loading"`